controllers/openstack_ansibleee_controller.go

```diff
@@ -137,6 +166,26 @@ func (r *OpenStackAnsibleEEReconciler) Reconcile(ctx context.Context, req ctrl.R
 		instance.Status.NetworkAttachments = map[string][]string{}
 	}
 
+	if instance.Status.Conditions.IsFalse(redhatcomv1alpha1.AnsibleExecutionJobReadyCondition) {
```

Comment:
if I understand correctly this means if an AnsibleEE CR reached Ready true state then it will never change that back to not ready. This implies that an AnsibleEE CR can only be used once to run the ansible tasks. If the client want to re-run it then it needs to create a new AnsibleEE CR. Am I correct? (I'm not saying it is wrong, I just try to understand the expected usage pattern of this CR)

Reply:
this what I understood from the current code:
```
1 - the job name is the CR name
2 - it tries to get the job
2.1 - if the job doesn't exist, create it
2.2 - if the job is already created, it will only get it
```
So I set the conditions according to it, if we want to re-run, we need to change the logic for handling the jobs

Reply:
if we need to rerun an AEE task we would delete and recreate the AEE CR

Reply:
OK,thanks for the explanation. Delete and re-create is OK for me in this context.